### ISSUE TITLE
Doesn't translate anymore

### ISSUE THREAD

**Issue by ZSnaider** [bbepis/XUnity.AutoTranslator#601]
Hello everyone. I translated the game "Alliance Alive hd remastered" using this program.
I use "XUnity.AutoTranslator-BepInEx-5.4.0.zip" and "BepInEx_win_x64_5.4.23.2.zip"
The game started to translate and for some time everything was fine. But when I started the game again, the translation of the new text disappeared. The program simply did not translate the text. When I pressed Alt+0, the program appeared - that means it was running and functioning. But why does it not translate?! The "read only" option was removed from the game folder - I even checked it through the command line. What should I do? Please help.

**Comment by Loli-Kyn** [comment]
I have the same problem. Maybe it's because I'm from Russia?

**Comment by Lmeks** [comment]
>  When I pressed Alt+0, the program appeared
Have you checked the status in the program window?

**Comment by Loli-Kyn** [comment]
> > When I pressed Alt+0, the program  appeared
> 
> Have you checked the status in the program window?
Yes. The screenshot shows my old work, but when I deleted the header, the new translation did not follow. 
![изображение_2024-12-03_223409448](https://github.com/user-attachments/assets/4fb80f93-088f-4a21-a9e6-a9d7f0c87cb0)

**Comment by Lmeks** [comment]
Is it like that for google translate v2 only?

**Comment by Loli-Kyn** [comment]
> Is it like that for google translate v2 only?

All of them. Now I tried to put the translator on a clean version of the game, and there does not work alt+o command, but alt+t and alt+1 command works. 
He doesn't respond to any other commands. I'm pretty sure the game is in Japanese
I used two versions of the translator in the video. The new one doesn't work and the old one has problems
https://youtu.be/I0-GNEF2LZg

**Comment by Loli-Kyn** [comment]
I don't see how to close the appeal. Please close it
